core/storage/user/gae_models_test.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+from core.platform import models
+from core.tests import test_utils
+
+(user_models,) = models.Registry.import_models([models.NAMES.user])
+
+
+class ExplorationUserDataModelTest(test_utils.GenericTestBase):
+    """Tests for the ExplorationUserDataModel class."""
+
+    DATETIME_OBJECT = datetime.datetime.strptime('2016-02-16', '%Y-%m-%d')
+    USER_ID = 'user_id'
+    EXP_ID_ONE = 'exp_id_one'
+    EXP_ID_TWO = 'exp_id_two'
+    MODEL_ID_ONE = '%s.%s' % (USER_ID, EXP_ID_ONE)
```

Comment:
These aren't really constants so I would be more inclined to compute them on the fly as they are needed rather than here.